### ISSUE TITLE
MicroPython support

### ISSUE THREAD

**Issue by Surya059** [tryolabs/norfair#39]
Hi, there are a lot of applications these days on embedded devices that need a customizable object tracking library like norfair. Is there any plan for supporting micropython?

**Comment by joaqo** [comment]
Hi! Thanks a lot for the interest. We want to make Norfair available on as many platforms as possible, though we don't know much about micropython. Does it support normal 3.6 python or is it a subset of the language?

Also, Norfair at the moment is just 100% plain python code. We barely have dependencies. The most troublesome ones for an embedded device I imagine would be Numpy and for sure OpenCV. We are in the process of removing OpenCV from the library though.

So to sum up, would plain python code + numpy run on micropython?

**Comment by joaqo** [comment]
Norfair now has a pure python version, so it should run in MicroPython 🎉

https://github.com/tryolabs/norfair/tree/master#installation
